.github/workflows/ci.yml

```diff
@@ -66,7 +67,7 @@ jobs:
       - name: Setup python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
```

Comment:
This bump is because, for some reason, combining under the different versions functions differently, 3.8 giving a wrong and lower result.